<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicEditor</class>
 <widget class="QDialog" name="GraphicEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>871</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrush">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Кисть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEraser">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ластик</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBackground">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменение фона</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddShape">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="brushSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="brushStyleComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="colorPicker">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Цвет</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="eraserSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeleteShape">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDrawName">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
